### DISCUSSION TITLE
Need help with git commit -m "Clean up ignored files"

### DISCUSSION THREAD

**Discussion by cj-rhoads** [community/community#27535] [New to GitHub]
Hello.  I'm relatively new to Github and not much of a techie.  
I've been using Github Desktop for more than a year to ensure the changes I make to the website get to my programmer.  I know how to set up a gitignore, but unfortunately, I messed up and set the git ignore too late.  So now I have several files (mostly .htaccess files) that should not be uploaded to the repository, but have already been tracked.  For months I've been "unchecking" them everytime I upload my legitimate changes and I'm getting tired of it.  I'd like to find a way to permanantly clean up these files that will, henceforth, be ignored because they've been set in the git ignore file.
I investigated and found these instructions:

```
Cleaning Ignored Files
In three steps, you can clean up your repository and make sure your ignored items are indeed ignored:
# Remove the files from the index (not the actual files in the working copy)
$ git rm -r --cached .
# Add these removals to the Staging Area...
$ git add .
# ...and commit them!
$ git commit -m "Clean up ignored files"

```
While it may sound simple to a techie, for me - I'm at a complete loss.  I take it that these would be commands that I would type in somewhere - but I do not know where.  I don't know how to launch Git Bash, and I don't have SSH connection set up anywhere and am not very good with terminal emulators like putty (though I have in the past, painstakingly, set them up and used them when necessary).  I don't really want to spend three hours downloading, setting up, and learning to use a command-driven version of Git Hub just so I can type in three commands.  Can anyone help me with a method for me to do this quickly and easily?

**Comment by CodyTolene** [comment]
Install Git [https://git-scm.com/download/win](https://git-scm.com/download/win) then you should have Git Bash. Now in Windows right click in the folder with the files you want to update (it needs to be part of a Git repository) and then select open Git Bash. Now you should be able to run the commands in the terminal.
